gemini/evaluation/evaluate_models_in_vertex_ai_studio_and_model_garden.ipynb

```diff
@@ -0,0 +1,2273 @@
+{
```

Comment:
<p>Adding some explanation of EvalTask here would be beneficial for practitioners. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>